language-plutus-core/cost-model-creation/CostModelCreation.hs

```diff
@@ -159,22 +198,22 @@ modInteger :: MonadR m => (SomeSEXP (Region m)) -> m (CostingFun ModelTwoArgumen
 modInteger = divideInteger
 
 lessThanInteger :: MonadR m => (SomeSEXP (Region m)) -> m (CostingFun ModelTwoArguments)
-lessThanInteger models = do
-  cpuModel <- readModelMinSize models "lessThanIntegerModel"
+lessThanInteger cpuModelR = do
+  cpuModel <- readModelMinSize cpuModelR
   pure $ CostingFun (ModelTwoArgumentsMinSize cpuModel) def
 greaterThanInteger :: MonadR m => (SomeSEXP (Region m)) -> m (CostingFun ModelTwoArguments)
 greaterThanInteger = lessThanInteger
```

Comment:
This kind of thing's a bit disconcerting at first glance because it looks like a cut and paste error! A remark (or a more neutral name for the common model) might be helpful.

Reply:
It's actually the same name for the collection of models as well as the individual models. I'll change.